src/Oracles/TestSettings.hs

```diff
@@ -0,0 +1,68 @@
+module Oracles.TestSettings (
+    TestSetting (..), testSetting, testRTSSettings
+    ) where
+
+import Hadrian.Oracles.TextFile
+import Base
+
+testConfigFile :: Action FilePath
```

Comment:
It would be great to add more comments here about the purpose of this module.

Please describe the purpose of the file `test/ghcconfig`, give a couple of examples of the settings that you read from it, and how they are used by the testsuite.

Reply:
Resloved